src/operators/insert.ts

```diff
@@ -0,0 +1,55 @@
+import { Operator, OperatorOptions, OperatorValidator } from '../operator';
+import { select } from '../selector';
+
+export interface InsertOperatorOptions extends OperatorOptions {
+  select?: string; // a value found using selection syntax
```

Comment:
It's pretty slick that we can mix in selection syntax wherever we need it. Go, us!

Reply:
I love this feature.  It has made the operator chain so much cleaner.